initScripts/aarch32/Ubuntu_16.04/Docker_18.03.sh

```diff
@@ -373,6 +364,24 @@ fetch_cexec() {
   popd
 }
 
+enable_32_bit_builds() {
+  __process_marker "Adding files to run 32-bit builds"
+
+  local temp_folder="/tmp/arm-linux-gnueabihf"
```

Comment:
A different approach to this will be to `docker create`, `docker cp` and finally `docker rm`. But, this approach also work.